cpp/unreal_plugins/SimulationController/Source/SimulationController/SphereAgent.cpp

```diff
@@ -199,13 +199,13 @@ std::map<std::string, ArrayDesc> SphereAgent::getObservationSpace() const
         observation_space["position"] = std::move(array_desc);
     }
 
-    if (Std::contains(observation_components, "rotation")) {
+    if (Std::contains(observation_components, "orientation")) {
```

Comment:
I want to use the term `rotation` and not `orientation` because that is what the Unreal functions are called, and I want to stick to Unreal's conventions unless we have a good reason not to. For the same reason, I'd like to use the term `location`, but I haven't got a chance to do that refactoring yet.

Reply:
Made these refactoring changes to `SphereAgent`, `VehicleAgent` and `CameraAgent`. Didn't do this to `UrdfBot` as it is not supported yet.

Reply:
Makes sense.